src/sentry/receivers/onboarding.py

```diff
@@ -423,8 +441,9 @@ def record_release_received(project, event, **kwargs):
     )
     if success:
         organization = Organization.objects.get_from_cache(id=project.organization_id)
-        owner_id = organization.default_owner_id
-        if not owner_id:
+        try:
+            owner: RpcUser = organization.get_default_owner()
+        except IndexError:
```

Comment:
Why are we changing this? The only difference between `default_owner_id` and `get_default_owner()` seems to be that get_default_owner does not raise an error.

Reply:
Ah, because we need the owner and not the id, nevermind.

Reply:
yes :)